diff -aurN colord-0.1.31.orig/rules/69-cd-sensors.rules.in colord-0.1.31/rules/69-cd-sensors.rules.in
--- colord-0.1.31.orig/rules/69-cd-sensors.rules.in	2013-03-24 12:46:10.271990757 +0100
+++ colord-0.1.31/rules/69-cd-sensors.rules.in	2013-03-24 12:48:25.603828538 +0100
@@ -86,8 +86,8 @@
 
 # color calibration device
 ENV{COLORD_SENSOR_KIND}=="*?", ENV{COLOR_MEASUREMENT_DEVICE}="1"
-ENV{COLORD_SENSOR_KIND}=="*?", ENV{ID_MODEL}=="", IMPORT{program}="usb_id --export %p"
-ENV{COLORD_SENSOR_KIND}=="*?", ENV{ID_MODEL_FROM_DATABASE}=="", IMPORT{program}="usb-db %p"
+ENV{COLORD_SENSOR_KIND}=="*?", ENV{ID_MODEL}=="", IMPORT{builtin}="usb_id"
+ENV{COLORD_SENSOR_KIND}=="*?", ENV{ID_MODEL_FROM_DATABASE}=="", IMPORT{builtin}="hwdb --subsystem=usb"
 
 # Allow the daemon to access the color devices
 ENV{COLORD_SENSOR_KIND}=="*?", GROUP="@daemon_user@"
